packages/backend/src/graphql/schema.graphql

```diff
@@ -40,6 +40,9 @@ type Query {
 
   "Fetch an Open Payments incoming payment"
   incomingPayment(id: String!): IncomingPayment
+
+  "Fetch a page of webhook events"
+  webhookEvents(input: GetWebhookEventsInput): WebhookEventsConnection
```

Comment:
`GetWebhookEventsInput` non-required because it'll just return the default pagination parameters, right?

Reply:
Yes that's the intention